bundler/onProdDone.js

```diff
@@ -0,0 +1,2 @@
+function onProdDone (stats) {}
```

Comment:
so this function gets replaced?

My question mark was just about whether you need a module containing just a noop function.